ePrint IACR.js

```diff
@@ -108,7 +107,13 @@ function doWeb(doc, url) {
 			ZU.processDocuments(articles, scrape);
 		});
 	} else {
-		scrape(doc, url)
+		if (url.search(/\.pdf$/)!= -1) {
+			//go to the landing page to scrape
+			//we use http to prevent (somewhat mysterious) same-origin violations leading to permission denied errors
+			url = url.replace(/\.pdf$/, "").replace(/^https/, "http")
```

Comment:
I don't have an idea why `http` is different from `https` here. If this hack is working, then this fine for me for the moment.